API/API.mli

```diff
@@ -1161,7 +1140,6 @@ end
 module UGraph :
 sig
   type t = UGraph.t
-  val pr_universes : (Univ.Level.t -> Pp.std_ppcmds) -> t -> Pp.std_ppcmds
```

Comment:
I would be in favor of updating `API/PROPERTIES` instead of removing this function.